Header header
int32 steering
int32 speed
float32 steering_angle
float32 v_radial
float64 radial_distance
geometry_msgs/Vector3 v_linear
geometry_msgs/Vector3 v_angular
